src/raft/uv.c

```diff
@@ -112,6 +113,12 @@ static int uvInit(struct raft_io *io, raft_id id, const char *address)
 	if (rv != 0) {
 		return rv;
 	}
+	md_format = metadata.format_version;
```

Comment:
[Very Nit]: If the variable is only going to live for ~5 lines then I would prefer to use `metadata.format_version` directly. If we move code in the future we do not have to worry about two variables.

Reply:
Yeah, I introduced this to avoid having to wrap the following lines, but it's probably better to just accept the wrapping :)